server.js

```diff
@@ -1,21 +1,81 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const UUID = require('uuid-js');
+const passport = require('passport');
+const Strategy = require('passport-facebook').Strategy;
+const cookieParser = require ('cookie-parser');
+const morgan = require ('morgan');
+const expressSession = require('express-session');
 const app = express();
 
+// Configure the Facebook strategy for use by Passport.
+//
+// OAuth 2.0-based strategies require a `verify` function which receives the
+// credential (`accessToken`) for accessing the Facebook API on the user's
+// behalf, along with the user's profile.  The function must invoke `cb`
+// with a user object, which will be set at `req.user` in route handlers after
+// authentication.
+passport.use(new Strategy({
+    clientID: '205751546505611',
+    clientSecret: 'e78708edd0c07fc11dcca84010cfc433',
+    callbackURL: 'http://localhost:3000/login/facebook/return',
```

Comment:
Need to find a way to pass the host true 'cause whenever we will deploy the app it will change